stdlib/int.mli

```diff
@@ -0,0 +1,139 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                         The OCaml programmers                          *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Integer values.
+
+    Integers are {!Sys.int_size} bits wide. All operations are taken
+    modulo 2{^[Sys.int_size]}. They do not fail on overflow.
```

Comment:
Might be worth mentioning the type uses 2's complement encoding to represent signed integers between min_int and max_int.  This is important to know for bitwise operations and the division.